.github/workflows/ci.yaml

```diff
@@ -75,3 +72,7 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           file: build/coverage-report.xml
+
+      - name: Check benchmarks
+        run: vendor/bin/phpbench run --revs=1 --iterations=1
```

Comment:
If we don't want to run the bench except for ensuring that it still works (not sure why it should not though, even in the future), wouldn't make more sense to have a separate workflow that can run every now and then?

Reply:
> not sure why it should not though, even in the future

All code that is not run regularly will rot and break

Reply:
> wouldn't make more sense to have a separate workflow that can run every now and then?

That would create a great overhead: separate CI job, separate setup, separate `composer install`... And for a step that [runs in less than a quarter of a second](https://github.com/getsentry/sentry-php/runs/4777675503?check_suite_focus=true#step:12:10), it seems an overkill.

Reply:
My suggestion was not related to the time spent running the step, but rather to the fact that if this step is here just to ensure that it runs, then it doesn't make sense to have it as part of the CI workflow

Reply:
> then it doesn't make sense to have it as part of the CI workflow

It's to make sure that upgrades to PHP, dependencies, etc. do not break it.

If we don't have it in CI to prevent rot, then having the benchmarks as part of the repository doesn't make sense either, as they will just become abandonware in a random directory ;-)

Reply:
With "CI workflow" I meant the workflow that runs for every push or for every PR. Do you think that running it once per week (just as an example) would still not be enough?

Reply:
:shrug: it's 0.25s, just let it run 

If it was a long process (that takes 30~40 minutes) then I understand running it on weekly basis, but running with every PR (especially when considering @dependabot) is necessary

Reply:
Ok then, let's keep it as-is